src/validation.h

```diff
@@ -92,6 +92,8 @@ static const unsigned int DEFAULT_CHECKLEVEL = 3;
 // one 128MB block file + added 15% undo data = 147MB greater for a total of 545MB
 // Setting the target to >= 550 MiB will make it likely we can respect the target.
 static const uint64_t MIN_DISK_SPACE_FOR_BLOCK_FILES = 550 * 1024 * 1024;
+/** Minimum size of a witness commitment structure. Defined in BIP 141. **/
+static const unsigned int MINIMUM_WITNESS_COMMITMENT = 38;
```

Comment:
```suggestion
static constexpr size_t MINIMUM_WITNESS_COMMITMENT{38};
```

The type where this is passed in our out seems to be `size_t`. See https://en.cppreference.com/w/cpp/container/vector/size and https://en.cppreference.com/w/cpp/container/vector/resize. So might as well use that type here.